callisto/notification/api.py

```diff
@@ -0,0 +1,95 @@
+import logging
+
+import gnupg
+import pytz
+
+from django.conf import settings
+from django.core.mail.message import EmailMultiAlternatives
+from django.utils import timezone
+
+from callisto.delivery.models import SentMatchReport
+from callisto.notification.models import EmailNotification
+
+logger = logging.getLogger(__name__)
+tzname = settings.REPORT_TIME_ZONE or 'America/Los_Angeles'
+timezone.activate(pytz.timezone(tzname))
+
+
+class NotificationApi(object):
```

Comment:
that make sense to me! i'm not 100% sure from looking at this how overrides would work but I think probably seeing it in action in the tests would be the quickest explanation :)